### PR TITLE
LPS-131936 Adjustments for categories labels

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
[Link to story or ticket](https://issues.liferay.com/browse/LPS-131936)

Proposed Solution
========
If a content have more than one category, we have added a Popover to show the rest of categories. We also have mode some style adjustments to the table to avoid horizontal scroll as much as posible. 

Change summary:
---------------
* LPS-131936 Add CategoriesPopover component
* LPS-131936 Change status column position
* LPS-131936 Improvements in content dashboard table columns width
* LPS-131936 Change content dashboard table styles
* LPS-131936 Translate text

Steps to Verify:
----------------
1. Add some web content or documents to a site
1. Create vocabularies with categories and assign them to the content just created
1. Go to Content Dashboard con configure it to show the vocabularies you just added
1. Check the table
1. If you want to try the categories popover with scroll you have to create a vocabulary with a lot os categories (like 15 categories) and assign all the categories to a web content or a document.

### CODE REVIEW

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Review by beltranrengifo — COMMENTED** [review]
Hi @claraizquierdo !
I see everything fine here 🎉 
However I wonder:
- Is there a clickoutside prop or something we can do to help close the popover by clicking outside, with ESC key, etc.?
- Is there a way to offset the popover position a few pixels from the trigger? I see It too close 
- Is a good idea to fixup the last commit with the first one? History can be a bit cleaner I think

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -178,4 +193,17 @@ html#{$cadmin-selector} {
 			padding-right: $sidebarWidth;
 		}
 	}
+
+	.categories-popover {
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:197]: Maybe we can add this style here to separate the popover from the trigger:
`transform: translateY(-8px);`

Comment (beltranrengifo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:197]: Also, is it a good idea to talk about this with Clay team? 🤔

Comment (beltranrengifo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:197]: Not that easy, I see 🤐

**Comment by claraizquierdo** [comment]
ci:forward

**Pull request closed without merging**